examples/neo-push/server/package.json

```diff
@@ -36,6 +36,7 @@
         "apollo-server-testing": "2.19.0",
         "cross-env": "7.0.3",
         "faker": "5.1.0",
+        "graphql-playground-middleware-express": "^1.7.22",
```

Comment:
Please could you pin this: 

```
"graphql-playground-middleware-express": "1.7.22",
```

Pining the dependencies means we reduce the chance of things breaking on version increments.